apps/portals/standards/src/pages/StandardsDetailsPage.tsx

```diff
@@ -28,7 +42,58 @@ export const standardDetailsPageContent: DetailsPageContentType = [
     id: 'About The Standard',
     title: 'About The Standard',
     element: (
-      <MarkdownSynapseFromColumnData columnName={'studyDescriptionLocation'} />
+      <DetailsPageContextConsumer columnName={'Name'}>
+        {({ value, context }) => {
+          const query: Query = {
+            sql: dataSql,
+            additionalFilters: getAdditionalFilters(
+              undefined,
+              { Name: value! },
+              ColumnSingleValueFilterOperator.EQUAL,
+            ),
+          }
```

Comment:
This is exactly how I think a query should be created in these cases, I think this looks great, though I don't think you'll need it in this specific case (based on other comments in this review). Thanks for switching to use this new pattern. I think we should look to update our other components to do this